tests/test_scripts.py

```diff
@@ -54,6 +55,21 @@ def test_load_into_counting():
     assert os.path.exists(outfile)
 
 
+def test_load_into_counting_nonwritable():
+    script = scriptpath('load-into-counting.py')
+    args = ['-x', '1e3', '-N', '2', '-k', '20', '-t']
+
+    outfile = utils.get_test_data('test-nonwritable')
+    os.chmod(outfile, stat.S_IWOTH | stat.S_IRUSR)
```

Comment:
Sure. Either create a new file and set its bits locally or store a file in git with the desired permissions.